python/cudf/cudf/io/orc.py

```diff
@@ -123,10 +123,11 @@ def read_orc_statistics(
         ValueError("URL content-encoding decompression is not supported")
 
     # Read in statistics and unpack
-    statistics = libcudf.orc.read_orc_statistics(filepath_or_buffer)
-    if not statistics:
-        return None
-    (column_names, raw_file_statistics, *raw_stripes_statistics,) = statistics
+    (
+        column_names,
+        raw_file_statistics,
+        raw_stripes_statistics,
```

Comment:
very minor but it seems you made a point of changing `stripes` to `stripe`, in a number of places, did you want to/forget to do it here as well?